python/sglang/srt/server_args.py

```diff
@@ -760,6 +761,12 @@ def add_cli_args(parser: argparse.ArgumentParser):
             action="store_true",
             help="Enable log prometheus metrics.",
         )
+        parser.add_argument(
+            "--kv-events-config",
+            type=str,
+            default=None,
+            help="The configurations for event publishing.",
```

Comment:
say this is used for NVIDIA dynamo framework

Reply:
Thanks @ByronHsu, updated the message.